requirements/dev.txt

```diff
@@ -9,8 +9,6 @@ anyio==3.7.1
     #   -c requirements/base.txt
     #   httpcore
     #   watchfiles
-appnope==0.1.4
```

Comment:
I am not sure where this change came from. It was done automatically by the requirements compiling command. It does seem safe, but happy to remove if we prefer.